todo_project/settings/base.py

```
@@ -54,6 +54,37 @@
+# JWT Verification Settings
+
+JWT_AUTH = {
+    "ALGORITHM": "RS256",
+    "PUBLIC_KEY": os.getenv("RDS_PUBLIC_KEY"),
+}
+
+# Cookie Settings
+
+JWT_COOKIE_SETTINGS = {
+    "RDS_SESSION_COOKIE_NAME": os.getenv("RDS_SESSION_COOKIE_NAME", "rds-session-development"),
+    "RDS_SESSION_V2_COOKIE_NAME": os.getenv("RDS_SESSION_V2_COOKIE_NAME", "rds-session-v2-development"),
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)